packages/helium-cli/emissions/iot.json

```diff
@@ -1,14 +1,14 @@
 [
   {
-    "startUnixTime": 1679064129,
-    "emissionsPerEpoch": "165616438356164"
+    "startTime": "2023-04-18T00:00:00Z",
+    "emissionsPerEpoch": "165,616,438.356164"
```

Comment:
`165,616,438.356164 * (100/93) * 365 = 65B` annualized for the first year

7% is held back here since there are no oracles yet.